wordpress-mobile/WordPress-Android
# Pass auth headers when copying remote media to pass through to Story composer

Fixes https://github.com/Automattic/stories-android/issues/694

We added a new method in WordPress Utils in https://github.com/wordpress-mobile/WordPress-Utils-Android/pull/79

To test:
1. use a private site
2. add a new story or add an empty story block in a post
3. tap on ADD MEDIA, and use the W section of the media picker, select media items there
4. observe these get copied and the Story composer is fed with the new media.
5. also repeat with media from Pexels (free photo library)

**:warning: NOTE**: once the WordPressUtils PR is merged, a new version should be tagged there and we should update the version pointed to by `build.gradle` in this PR  (see commit fd2e702767) to the actual tag on WordPressUtil repo.

## Regression Notes
1. Potential unintended areas of impact
N/A - this is a new method that will only run when authentication headers are available

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A, the tests are not at the Utils level, but at the UseCase level, which remains the same

PR submission checklist:
